### PR TITLE
Ravendb-11249 ResetServer command modified to not open browser and changed to restartServer

### PR DESCRIPTION
**Pull request by haludi**
https://issues.hibernatingrhinos.com/issue/RavenDB-11249

### CODE REVIEW

File: src/Raven.Server/Utils/Cli/RavenCli.cs [hunk 1]
```diff
@@ -1133,7 +1133,7 @@ private static bool CommandHelp(List<string> args, RavenCli cli)
                 new[] {"script <server|database> [database]", "Execute script on server or specified database. WARNING: Use with care!"},
                 new[] {"logout", "Logout (applicable only on piped connection)"},
                 new[] {"openBrowser", "Open the RavenDB Studio using the default browser"},
-                new[] {"resetServer", "Restarts the server (shutdown and re-run)"},
+                new[] {"restartServer", "Restarts the server (shutdown and re-run)"},
```
Comment (ayende) [inline src/Raven.Server/Utils/Cli/RavenCli.cs]: You have to also have the old command functional.

Comment (haludi) [reply src/Raven.Server/Utils/Cli/RavenCli.cs]: For back compatibility?

Comment (ayende) [reply src/Raven.Server/Utils/Cli/RavenCli.cs]: yes

Comment (haludi) [reply src/Raven.Server/Utils/Cli/RavenCli.cs]: So, I'll modify the implementation to accept `resetServer`
and what about the description? to describe the old version too? (same question about the documentation)

Comment (ayende) [reply src/Raven.Server/Utils/Cli/RavenCli.cs]: Both should use the same description

**Review by ppekrol — APPROVED** [review]

**Review by ayende — APPROVED** [review]

**Pull request merged**